.circleci/config.yml

```
@@ -449,27 +463,20 @@ workflows:
       # and now it fails with stack too deep after https://github.com/ethereum-optimism/superchain-ops/pull/528.
       # We wll need to rewrite the rehearsals with the new superchain-ops structure anyway, so this is ok.
       # - just_simulate_sc_rehearsal_1
-      
-      - just_simulate_nested:
-          task: "/eth/ink-003-fp-holocene-pectra-upgrade"
-          context:
-            - circleci-repo-readonly-authenticated-github-token
 
       - just_simulate_sc_rehearsal_2:
           context:
             - circleci-repo-readonly-authenticated-github-token
       - just_simulate_sc_rehearsal_4:
           context:
             - circleci-repo-readonly-authenticated-github-token
-      - just_simulate_nested:
-          task: "eth/031-op-mainnet-pectra-defense"
-          context:
-            - circleci-repo-readonly-authenticated-github-token
 
+      # Simulation sequence for Superchain Version Sync (at the 27th of February 2025). The next task would be 31 but we need to wait until the task is 31 is merged.
+      # tasks: "030 031"
       - simulate_sequence:
           name: simulate_sequence_eth
           network: "eth"
-          tasks: "ink-003"
+          tasks: "unichain-001"
```

here, this is the whole sequence you need to simulate so first ink and then uni task like this: 
```suggestion
          tasks: "ink-003 unichain-001"
```